public/gameInfo.php

```diff
@@ -89,6 +91,27 @@
 ?>
 <?php if ($gate): ?>
     <?php RenderContentStart($pageTitle) ?>
+    <script>
+        /**
+         * @param {number} currentPreferenceValue
+         * @param {number} gameId
+         */
+        function disableMatureContentWarningPreference(currentPreferenceValue, gameId) {
```

Comment:
currentPreferenceValue is no longer used, and gameId could be eliminated if you change line 111 to:
```
window.location = `/game/<?= $gameId ?>`;
```

Reply:
Fixed